pymdownx/snippets.py

```diff
@@ -123,6 +124,8 @@ def parse_snippets(self, lines, file_name=None):
                         pass
                     if file_name:
                         self.seen.remove(file_name)
+                elif not os.path.exists(snippet) and self.check_paths:

```

Comment:
Should be: `elif snippet and self.check_paths:`. If we have a non-empty string for `snippet`, we know we've already tested if it exists, and since it didn't exist, we got here.  So just check that `snippet` is not empty and that `self.check_paths` is enabled.

Reply:
+1, fixed